app/presenters/content_item_subscription_presenter.rb

```diff
@@ -3,6 +3,14 @@ def initialize(content_item)
     @content_item = content_item
   end
 
+  def title
+    if content_item["content_id"] == "c4cd0e8a-3ae1-4385-8936-1cfafe5031fb"
+      "Coronavirus (COVID-19)"
```

Comment:
What is the reason for overriding it rather than changing it? We want to keep displaying the original title everywhere else, and change it here only?